winui/TreeGrid/Export-To-Excel.md

```diff
@@ -0,0 +1,595 @@
+
+You can customize the rows based on the record values by using the `CellsExportHandler`.
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+options.CellsExportHandler = CellsExportHandler;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+
+private void CellsExportHandler(object sender, TreeGridCellExcelExportOptions e)
+{
+    if ((e.Node == null))
+        return;
+    var record = e.Node as EmployeeInfo;
+
+    if (record.City == "US")
+    {
+        e.Range.CellStyle.Color = Color.LightYellow;
+        e.Range.CellStyle.Font.Color = ExcelKnownColors.Pink;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-To-Excel_images/winui-treegrid-row-style.png" alt="Customizing Row Style based on Data while Exporting to Excel in WinUI TreeGrid" width="100%" Height="Auto"/>
+
+Here, the records having the `City` name as `US` are customized.
+
+### Customize the cells based on column name
+
+You can customize the cells based on the [GridCellExcelExportingEventArgs.ColumnName](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.Converter.GridCellExcelExportingEventArgs.html#Syncfusion_UI_Xaml_Grid_Converter_GridCellExcelExportingEventArgs_ColumnName) property in [CellsExportingEventHandler](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.TreeGrid.Converter.TreeGridExcelExportingOptions.html#Syncfusion_UI_Xaml_TreeGrid_Converter_TreeGridExcelExportingOptions_CellsExportingEventHandler).
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+options.CellsExportHandler = CellsExportHandler;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+
+private void CellsExportHandler(object sender, TreeGridCellExcelExportOptions e)
+{
+    if (e.ColumnName != "ID")
+        return;
+
+    e.Range.CellStyle.Font.Size = 12;
+    e.Range.CellStyle.Font.Color = ExcelKnownColors.Pink;
+    e.Range.CellStyle.Font.FontName = "Segoe UI";
+}
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-To-Excel_images/winui-treegrid-column-cell-customization.png" alt="Customizing Cell Values based on Column name while Exporting to Excel in WinUI TreeGrid" width="100%" Height="Auto"/>
+
+Here, the `ID` column cells are customized while exporting.
+
+## Customize exported workbook and worksheet
+
+The TreeGrid can be exported to Excel using [XlsIO](http://help.syncfusion.com/file-formats/xlsio/overview). You can refer to the [XlsIO documentation](http://help.syncfusion.com/file-formats/xlsio/working-with-excel-worksheet) for manipulating the workbooks and sheets after exporting.

```

Comment:
The TreeGrid can be exported to Excel using the [XlsIO](http://help.syncfusion.com/file-formats/xlsio/overview). Refer to the [XlsIO documentation](http://help.syncfusion.com/file-formats/xlsio/working-with-excel-worksheet) for manipulating the workbooks and sheets after exporting.

Reply:
Updated